packages/feedback-service/src/feedbacks/schema.ts

```diff
@@ -15,13 +15,11 @@ export const FeedbackSchema: Schema = new Schema({
     path: { type: String },
   },
   category: { type: String, enum: ['BUG', 'FEEDBACK'] },
-  createdOn: { type: Date, default: Date.now },
   createdBy: String,
-  updatedOn: { type: Date, default: Date.now },
   updatedBy: String,
-});
+}, { timestamps: { createdAt: 'createdOn', updatedAt: 'updated_On' } });
```

Comment:
```suggestion
}, { timestamps: { createdAt: 'createdOn', updatedAt: 'updated_On' } });
```

Type error for updated_On

Reply:
Done